pyaedt/modeler/Primitives.py

```diff
@@ -1663,7 +1665,11 @@ def _refresh_all_ids_from_aedt_file(self):
 
             if o.analysis_type:
                 o._solve_inside = attribs['SolveInside']
-                o._material_name = attribs['MaterialValue'][1:-1]
+                # o._material_name = attribs['MaterialValue'][1:-1]
+                if 'MaterialValue' in attribs:
+                    o._material_name = attribs['MaterialValue'][1:-1]
```

Comment:
Are you sure that we don't need to remove the " before and after the material name once we get it?

Reply:
Sure, it is formatted differently.